Resources/translations/messages.fr.xliff

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
```

Comment:
You should name this file with bundle name so it's not going to conflict with project translation and it ease overrides.